psydac/feec/multipatch/non_matching_operators.py

```diff
@@ -0,0 +1,1238 @@
+import os
+import numpy as np
+from scipy.sparse import eye as sparse_eye
+from scipy.sparse import csr_matrix
+from sympde.topology import Boundary, Interface
+from psydac.fem.splines import SplineSpace
+from psydac.utilities.quadratures import gauss_legendre
+from psydac.core.bsplines import breakpoints, quadrature_grid, basis_ders_on_quad_grid, find_spans, elements_spans
+from copy import deepcopy
```

Comment:
According to PEP-8, the module imports should be organized in this order:
1. Modules from the Python standard library
2. Third-party modules
3. Other modules from the current library (Psydac)

The three groups should be separated by a blank line. Additionally I would like to separate our own libraries (Pyccel and SymPDE) from the other third-party libraries.

```suggestion
from copy import deepcopy

import numpy as np
from scipy.sparse import eye as sparse_eye
from scipy.sparse import csr_matrix

from sympde.topology import Boundary, Interface

from psydac.fem.splines import SplineSpace
from psydac.utilities.quadratures import gauss_legendre
from psydac.core.bsplines import breakpoints, quadrature_grid, basis_ders_on_quad_grid, find_spans, elements_spans
```